.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           node-version: 16
 
       - name: Install dependencies
-        run: npm ci
```

Comment:
The `npm ci` command is reliant on a package-lock file, while `npm install` is not. Switching over to `npm install` so we can ignore the package-lock file without breaking the build action.

Reply:
awesome! I love it!